rue89-tech/edx-theme
# Remove $static-path to make theme work with Hawthorn

This PR removes `$static-path` definitions from the theme's Sass files as these definitions prevented `collectstatic` command  of `open-release/hawthorn.beta1` branch of `edx-platform` from working.
It also adds recompiled css files.

**Ticket:** [OC-4641](https://tasks.opencraft.com/browse/OC-4641)
**Testing:**
1. Visit https://rue89-hawthorn.opencraft.hosting/ and assure it's up (meaning that the provisioning went well :))
1. See if theming of https://rue89-hawthorn.opencraft.hosting/ looks ok.

**Reviewer:**
Signed-off-by: Tomasz Gargas <tomasz@opencraft.com>
